src/beautifiers/eslint.coffee

```diff
@@ -23,12 +24,23 @@ module.exports = class ESLintFixer extends Beautifier
       allowUnsafeNewFunction ->
         importPath = Path.join(projectPath, 'node_modules', 'eslint')
         try
-          CLIEngine = require(importPath).CLIEngine
+          importPath = Path.join(projectPath, 'node_modules', 'eslint')
+          context.which('eslint').then((globalEslintPath) ->
+            if `globalEslintPath !== 'eslint'`
+              importPath = Path.resolve(globalEslintPath, '../../lib/node_modules/eslint')
+            else
+              context.deprecate 'Error: it seems like you did not install eslint globally, try to run npm i -g eslint'
+            CLIEngine = require(importPath).CLIEngine
 
-          cli = new CLIEngine(fix: true, cwd: projectPath)
-          result = cli.executeOnText(text).results[0]
-
-          resolve result.output
+            cli = new CLIEngine(fix: true, cwd: projectPath)
+            result = cli.executeOnText(text).results[0]
+            if not result.output
```

Comment:
What does this block do exactly?

Reply:
Here is the case: There is a `.eslintrc` config file in the `projectPath`,  and there are some `async/await` syntax in the current js code which input by `text` param. However, I did not set `ecmaVersion: 8` in my `.eslintrc` so that the `result` of `cli.executeOnText(text).results[0]` will get `undefined`.

> Reference:  [eslint project issue](https://github.com/eslint/eslint/issues/8126)

In the current logic of eslint-beautifier from the master branch, there is not any warning or error notification for user, it just get nothing. So these logic is a friendly error message for user.